### ISSUE TITLE
Users can reply to Bot

### ISSUE THREAD

**Issue by MirianaRebeggiani** [OfficeDev/microsoft-teams-apps-company-communicator#530]
Using the MS Teams Deskop app (Microsoft Teams Version 1.4.00.26376 (64-bit)) (WD 10 build 19043) users can reply to Bot:
![image](https://user-images.githubusercontent.com/28442653/139070049-c4ee6dad-44a2-40a6-8b69-aa81f11d09e9.png)

I've check the manifest and it has the parameter isNotificationOnly: true.

Moreover if I use the MS Teams Web it works correctly and I can't send messages to bot.

**Comment by SubbaReddi** [comment]
It is issue in Desktop teams client. If the app is added under "Installed apps" in app setup policy, then this this issue occuring

**Comment by brownt63** [comment]
> It is issue in Desktop teams client. If the app is added under "Installed apps" in app setup policy, then this this issue occurring

@SubbaReddi  How are you suppose to deploy this app to all users in your enterprise if you can't add the app under "Installed apps" in the global app setup policy.  I don't think this is the issue, if so why is this the problem?

**Comment by brownt63** [comment]
> > It is issue in Desktop teams client. If the app is added under "Installed apps" in app setup policy, then this this issue occuring
> 
> Thank you! Now I've tried to remove from app setup policy the app and I'm waiting for the Cloud replication to check if this resolves the issue.

@MirianaRebeggiani  Have you had any success with removing it?  If so, how are you going to deploy this app to new users in your organization going forward?

**Comment by SubbaReddi** [comment]
@brownt63 : If it is company communicator, you have option of proactive installation as mentioned [here](https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/wiki/Deployment-guide#2-deploy-to-your-azure-subscription)
As it is Product related bug, fix will be done.

**Comment by brownt63** [comment]
> @brownt63 : If it is company communicator, you have option of proactive installation as mentioned [here](https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/wiki/Deployment-guide#2-deploy-to-your-azure-subscription) As it is Product related bug, fix will be done.

@SubbaReddi OK.  Thanks for the reply back!  Any possible ETA as to when this will be fixed?  We already have it in our app setup policy, so all the users already have it installed, so it's too late for existing users  :(

**Comment by brownt63** [comment]
When will this get fixed?

**Comment by Porkai-Pandian** [comment]
Hi @brownt63,

Can you please try the below work around and let us know if it resolved your issue.

_In users manifest.json file, remove ‘messageTeamMembers’ text from ‘permissions’ section, this will disable the option for users to reply to messages_

**Comment by brownt63** [comment]
No, the "Reply" option still shows in the menu when you right click on a message.  Even after waiting and clearing teams cache.

**Comment by MirianaRebeggiani** [comment]
Hi, @brownt63  the workaround suggested by @SubbaReddi it seems work, but unfortunatly there is another bug related to the chat of the bot. 
If you remove from installation app "the company communicator" the chatbox is disabled but if you try to block and unblock bot conversation then you can send messages.

**Comment by brownt63** [comment]
Hi, @MirianaRebeggiani OK, so that's now 2 bugs at this point.  I'll have to wait until a fix is done for all things then. I can't make those kind of changes and still not get desired results for all enterprise users.

Anyone know of an ETA as to when all will get fixed?

**Comment by brownt63** [comment]
@faalive  I had a case open for months on this issue and they told me there was nothing they could do and that I had to file my issue here only for support of it!

**Comment by faalive** [comment]
Last time I have received "support" because this issue was appear toghether with teams fix update; once that teams error has been correct is disappear also opportunity to send message. Now where is the difference?

**Comment by magurrapu** [comment]
@faalive @brownt63 @MirianaRebeggiani The code fix is development stage. We are working on it to resolve the issue asap. ETA Monday 11/15/21

**Comment by faalive** [comment]
> @faalive @brownt63 @MirianaRebeggiani The code fix is in development stage. We are working on it to resolve the issue asap.

We have an idea about timing? I don't want force but just to have something to say to customers.
Regards

**Comment by magurrapu** [comment]
@faalive The fix is in our development backlog, we will take up this based on priority and right now we do not have any ETA.

Regards,
Machander

**Comment by faalive** [comment]
> @faalive The fix is in our development backlog, we will take up this based on priority and right now we do not have any ETA.
> 
> Regards, Machander

OK, if there's a process to rise severity about this, please give me info because I have 125k Platform seat blocked that represent 50% of Italy pipe and 30% of EMEA pipe!
Regards

**Comment by cristianoag** [comment]
Since the last Teams client update all deployments are allowing replies to the bot messages. Before the client update we had deployments where the problem was not happening and deployments where users were able to reply to bot messages. Now everybody can reply.

**Comment by gsv022** [comment]
Hi @MirianaRebeggiani,
We are closing this issue request and please feel free to reach out to us on the deployment or functionality issues of Company Communicator app by logging the new issue.
